crates/catalog/rest/src/catalog.rs

```diff
@@ -586,16 +648,46 @@ mod _serde {
         pub(super) metadata: TableMetadata,
         pub(super) config: Option<HashMap<String, String>>,
     }
+
+    #[derive(Debug, Serialize, Deserialize)]
+    #[serde(rename_all = "kebab-case")]
+    pub(super) struct CreateTableRequest {
+        pub(super) name: String,
+        pub(super) location: Option<String>,
+        pub(super) schema: Schema,
+        pub(super) partition_spec: Option<PartitionSpec>,
+        pub(super) write_order: Option<SortOrder>,
+        pub(super) stage_create: bool,
```

Comment:
https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.py#L764
```suggestion
        pub(super) stage_create: Option<bool>,
```